docs/tutorials/get-started/try-viam-sdk.md

```diff
@@ -38,39 +38,108 @@ If you are running out of time during your rental, you can [extend your rover re
+
+Next, create a file named <file>square.go</file> and paste the boilerplate code from the **code sample** tab of the Viam app into your file.
+Then, save your file.
+
+Initialize your project, and install the necessary libraries, and then run the program to verify that the Viam SDK is properly installed and that the `viam-server` instance on your robot is live:
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
 go mod init square
 go mod tidy
 go run square.go
 ```
 
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+The easiest way to get started writing an application with Viam is to navigate to the [robot page on the Viam app](https://app.viam.com/robots), select the **code sample** tab, then select **TypeScript** and copy the boilerplate code.
+
+This code snippet imports all the necessary libraries and sets up a connection with the Viam app in the cloud.
+
+Next, create a file named <file>main.ts</file> and paste the boilerplate code from the **code sample** tab of the Viam app into your file.
+Then, save your file.
+
+Create another file named <file>package.json</file> with the following contents:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "name": "test-rover",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "esbuild ./main.ts --bundle --outfile=static/main.js --servedir=static",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Viam Docs Team",
+  "license": "ISC",
+  "devDependencies": {
+    "esbuild": "0.16.12"
+  },
+  "dependencies": {
+    "@viamrobotics/sdk": "*"
+  }
+}
+```
+
+Create a folder <file>static</file> and inside it another file named <file>index.html</file>.
+Add the following markup:
+
+```html {class="line-numbers linkable-line-numbers"}
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Drive a Viam Rover</title>
+    <link rel="icon" href="favicon.ico" />
+  </head>
+  <body>
+    <div id="main">
+      <button id="main-button" disabled=true>
+        Click me
+      </button>
+    </div>
+    <script type="module" src="main.js">
+    </script>
+  </body>
+</html>
+```
+
+Run the following commands to install the necessary libraries, and then run the program to verify that the Viam SDK is properly installed and that the `viam-server` instance on your robot is live:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+npm install
+npm start
+```
+
 {{% /tab %}}
 {{< /tabs >}}
 
-If you successfully configured your robot and it is able to connect to the Viam app, the program you ran prints the names of your rover's resources to the terminal.
+If you successfully configured your robot and it is able to connect to the Viam app, the program prints some output to the terminal, including the names of your rover's resources.
 These are the components and services that the roobot is configured with in the Viam app.
```

Comment:
Since the SDK is 100% browser based, this needs to change a bit - feel free reword this but here's the gist of what actually happens.

Also, there was a typo - I changed roobot to robot (just want to make sure that doesn't get lost in the diff if you end up discarding it).

```suggestion
Open a web browser and visit `localhost:8000`. You should see a disabled button that says `Click me`. If you successfully configured your robot and it is able to connect to the Viam app, the button will become enabled. If you open the developer console, you should see some output including the names of your rover's resources.
These are the components and services that the robot is configured with in the Viam app.
```

Reply:
good catch - thank you!